### PR TITLE
Widgets: LangSelector: Sort languages model by name

### LINKED ISSUE #283: Languages list is sorted according to English name of language even when the system language is not English
The lists of languages in the list boxes are always sorted according to the language names in English.

When the system language is different (i.e. Hebrew), the names of the languages are properly translated to it, but are not re-sorted accordingly. This makes it difficult to find the desired language in the list.

The language list boxes should be sorted according to the values that they actually show.

Reported downstream [here](https://bugs.launchpad.net/ubuntu/+source/dialect/+bug/1990759).

### PR DESCRIPTION
**Pull request by rafaelmardojai**
Fixes #283

### CODE REVIEW

File: dialect/widgets/lang_selector.py [hunk 1]
```diff
@@ -105,10 +107,15 @@ def _popover_closed(self, _popover):
     def _create_lang_row(self, lang):
         return LangRow(lang)
 
-    def _filter_func(self, item):
+    def _filter_langs(self, item):
         search = self.search.get_text()
         return bool(re.search(search, item.name, re.IGNORECASE))
 
+    def _sort_langs(self, lang_a, lang_b, _data):
+        a = lang_a.name.lower()
```
Comment (fushinari) [inline dialect/widgets/lang_selector.py:115]: How does this work with non-latin languages? Any idea?

Comment (rafaelmardojai) [reply dialect/widgets/lang_selector.py:115]: Technically it's comparing against the numbers behind the string characters, and unicodes are ordered alphabetically so should work for all alphabets. But let me try testing it.

Comment (rafaelmardojai) [reply dialect/widgets/lang_selector.py:115]: Yeah, tested it and seems to be working nicely with other alphabets.

Comment (fushinari) [reply dialect/widgets/lang_selector.py:115]: Okay, cool!

**Pull request merged**